apps/safe-claiming-app/package.json

```diff
@@ -5,19 +5,19 @@
   "homepage": "/safe-claiming-app",
   "dependencies": {
```

Comment:
do we want to update the dependencies in the Claiming app in this PR?

Reply:
We shouldn't but they are all minor dependencies, shouldn't cause issues